lib/talkingscoreslib.py

```diff
@@ -14,8 +14,11 @@
 us['warnings'] = 0
 from abc import ABCMeta, abstractmethod
 from jinja2 import Template
-from _datetime import datetime
+from datetime import datetime #start_time = datetime.now()
 logger = logging.getLogger("TSScore")
+import time
+
+global settings
```

Comment:
Is this required to access options from the Django settings?  did you already consider using an import rather than relying on global scope?

Reply:
During HTMLTalkingScoreFormatter.__init__() - It reads in the options that the user set for this talking score and stores them in settings.  But later on eg when it is rendering each individual note - it needs to know whether to put the dot before or after the rhythm...  Or similar question for the octave description / tempo etc.  I did wonder if the fact that I was trying to make something global meant there was a better way to do it?  I could pass relevant settings down to the notes etc?